common/app/controllers/EmailSignupController.scala

```diff
@@ -78,13 +79,14 @@ class EmailSignupController(wsClient: WSClient, val controllerComponents: Contro
   }
 
   def renderForm(emailType: String, listId: Int): Action[AnyContent] = Action { implicit request =>
-    Cached(1.day)(RevalidatableResult.Ok(views.html.emailFragment(emailLandingPage, emailType, listId)))
+    val identityName = EmailNewsletter(listId).map(_.identityName).getOrElse("today-uk")
```

Comment:
Is it ok to default to `today-uk`? Could there be a case where we are showing a page describing a different newsletter and then we subscribe them to `today-uk`?